x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/components/deduplication_panel/deduplication_panel.scss

```diff
@@ -2,13 +2,13 @@
   .selectableWrapper {
     padding: 1rem;
     border-radius: .675rem;
```

Comment:
Sorry to nit, but we should be using variables here as well 😉 

`.675 rem` calculates out to about 10.8px. Is that value right?

Reply:
😅  I've fixed both of these in e24984c.  I'm using `$euiSizeM * .675` as the value for `border-radius` to match [how EUI's codebase defines their own sizes](https://github.com/elastic/eui/blob/e43e5851ac682287b1d6d0e3ed28840f76715cd6/src/global_styling/variables/_size.scss).